glossary.yml

```
@@ -4188,7 +4188,11 @@
     def: >
       A collection of code, data, and documentation that can be distributed and
       re-used. Also referred to in some languages as a [library](#library) or [module](#module).
-
+  ja:
+    term: パッケージ
+    def: >
+      持つ関数のまとまりとデーターと文書化が分散して、再使えるように環境に加えること出来のこと。
+      それに、何かの言語の場合は「[ライブラリー](#library)」と「[モジュール](#module)」と読んでいのこと。
```

Suggestion: 「[ライブラリー](#library)」もしくは「[モジュール](#module)」と呼ぶプログラミング言語もある。

---

Thanks for the feedback. I think this better reflects the meaning of the English version. The terms aren't exclusive, different languages use "package", "library", "module", "plug-in", "add-on", "extension", etc.

```suggestion
      それに、プログラミング言語のパッケージも「[ライブラリー](#library)」や「[モジュール](#module)」と呼ぶもある。
```